skyportal/handlers/api/photometry.py

```diff
@@ -337,18 +338,104 @@ def post(self):
         df['standardized_flux'] = standardized.flux
         df['standardized_fluxerr'] = standardized.fluxerr
 
+        df["hash"] = df.apply(
+            lambda row: hash(
+                (
+                    row["instrument_id"],
+                    row["filter"],
+                    row["mjd"],
+                    row["standardized_flux"],
```

Comment:
I think we should hash on input user data rather than data standardized against some magnitude system. If the flux standards for a particular magnitude system get updated in sncosmo, these standardized values may change for the same input data.

Reply:
e.g., I use a new version of the AlphaLyr HST calspec standard spectrum